config-service-impl/src/main/java/org/hypertrace/config/service/store/DocumentConfigStore.java

```diff
@@ -116,9 +136,13 @@ public Value getConfig(ConfigResource configResource) throws IOException {
       Value config = configDocument.getConfig();
       if (seenContexts.add(context) && !ConfigServiceUtils.isNull(config)) {
         contextSpecificConfigList
-            .add(ContextSpecificConfig.newBuilder().setContext(context).setConfig(config).build());
+            .add(ContextSpecificConfig.newBuilder().setContext(context).setConfig(config)
```

Comment:
could probably pull the `ConfigDocument -> ContextSpecificConfig` into its own function in use in the two spots

Reply:
updated it